plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/TestKuduConnectorTest.java

```diff
@@ -731,37 +729,24 @@ public void testWrittenStats()
 
     @Test
     @Override
-    public void testCreateTableAsSelectNegativeDate()
+    public void testVarcharCastToDateInPredicate()
     {
-        // Map date column type to varchar
-        String tableName = "negative_date_" + randomNameSuffix();
+        assertThatThrownBy(super::testVarcharCastToDateInPredicate)
+                .hasStackTraceContaining("Table partitioning must be specified using setRangePartitionColumns or addHashPartitions");
 
-        try {
-            assertUpdate(format("CREATE TABLE %s AS SELECT DATE '-0001-01-01' AS dt", tableName), 1);
-            assertQuery("SELECT * FROM " + tableName, "VALUES '-0001-01-01'");
-            assertQuery(format("SELECT * FROM %s WHERE dt = '-0001-01-01'", tableName), "VALUES '-0001-01-01'");
-        }
-        finally {
-            assertUpdate("DROP TABLE IF EXISTS " + tableName);
-        }
+        abort("TODO: implement the test for Kudu");
     }
 
     @Test
     @Override
+    @SuppressWarnings("deprecation")
```

Comment:
@ebyhr Is there a better way to solve it without adding this annotation ? For other connector tests like MariaDB we don't have this annotation -

Reply:
The reason is Kudu connector enables `air.compiler.fail-warnings` unlike MariaDB connector. I don't come with a better way.

Reply:
So do we need to disable the flag or keep the annotation ? I would vote for keeping the annotation as once we throw a valid trino failure then the annotation can be removed. WDYT ?

Reply:
I have the same opinion - keeping the annotation in this PR.